qdev_wrappers/T12/merlin/timeinstrument.py

```diff
@@ -0,0 +1,23 @@
+import time
+# import datetime
+from qcodes import Instrument
+
+
+class TimeInstrument(Instrument):
+    def __init__(self, name, **kwargs):
+        super().__init__(name, **kwargs)
+
+        self.add_parameter('time',
+                           unit='s',
+                           get_cmd=time.time,
```

Comment:
Just a cautious warning: we have been experiencing problems with `time.time` as it does not guarantee you any precision, it turns out that when calling it consecutively you don't necessarily get a monotonous result. As an alternative there is the perf_counter.